src/app/components/dynamic-hub/dynamic-hub.component.html

```diff
@@ -17,7 +17,19 @@
     <div class="dynamic-hub__content">
         <div class="dashboards">
             <div class="dashboard" *ngFor="let data of cards | keyvalue">
-                <h2>{{data.key | titlecase | translate}} {{'dashboards' | translate}}</h2>
+                <h2>
+                    <span class="dashboard-title">{{data.key | titlecase | translate}} {{'dashboards' | translate}}</span>
+                    <ng-container *ngIf="!!canCreateDashboardByOrg.get(data.key)">
```

Comment:
you can use the getValue pipe that is declared in web components

Reply:
Done